src/client/delphi_epidata.py

```diff
@@ -20,684 +20,731 @@
 # preference, even if you've installed it and then use this script independently
 # by accident.
 try:
-  _version = get_distribution('delphi-epidata').version
+    _version = get_distribution("delphi-epidata").version
 except DistributionNotFound:
-  _version = "0.script"
+    _version = "0.script"
 
-_HEADERS = {
-  "user-agent": "delphi_epidata/" + _version + " (Python)"
-}
+_HEADERS = {"user-agent": "delphi_epidata/" + _version + " (Python)"}
+
+
+class EpidataException(Exception):
+    ...
```

Comment:
this looks like more than just a run through black!  at first i was quite impressed thinking that black did this automatically, but i came to my senses pretty quickly...

i think `pass` is a better token to use here because its not all punctuation, and i think its intent is a little clearer.  however, ymmv and its fine to leave it as-is.
```suggestion
    pass
```

Reply:
blackGPT in 5 years probably will, but ye

and sure, i don't have a strong preference here.